docs/get-started/v2/get-started/features/recordings/live-stream-recording.mdx

```diff
@@ -0,0 +1,143 @@
+        1. **Unzipped** - This is chunked upload (M3U8) and is immediately playable.
+        2. **Zipped** - This is M3U8 file zipped and uploaded, which is optimised for storage. 
+
+The following video will do a walkthrough of the settings.
+
+<br/>
+<video loop="true" controls="controls" id="vid">
+    <source src="https://storage.googleapis.com/100ms-cms-prod/cms/enable_vod_recording_l_aa16608ff0/enable_vod_recording_l_aa16608ff0.mp4?updated_at=2023-08-24T07:09:42.167Z" type="video/mp4" />
+</video>
+<br/>
+
+> **Note** 
+>
+> You can also configure recording settings through the Policy API or provide a runtime configuration using the Live streams API.  
+
+To understand the difference between 100ms hosted and non-hosted storage, and how it affects your VOD recording upload settings, you can refer to the table below.
+
+|                                                       | 100ms Hosted                                                                                        | User Cloud Bucket (Non-100ms)                                                    |
+|:------------------------------------------------------|:----------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------|
+| Default Template Configuration (At template creation) | Unzipped M3U8                                                                                       | Unzipped M3U8                                                                    |
+| Zipped Recording (M3U8) Upload                        | Not Available                                                                                       | Available                                                                        |
+| Unzipped Recording (M3U8) Upload                      | Available                                                                                           | Available                                                                        |
+| Direct VOD Playback (Via CDN)                         | Yes, we host the recordings and provide you with CDN URL directly through our Recording Assets API. | No, you are required to set up your own CDN servers to allow on-demand playback. |
+
+> **Important Note - Zipped Recording Files**
+>
+> - For all the templates created before 27th August 2023, the M3U8 file is zipped and uploaded, irrespective of the storage you've configured. 
+> - For older templates with recordings hosted on 100ms, you have an option to switch from Zipped to Unzipped recording files upload, but you cannot switch back once you've saved that setting.
+> - If you've hosted the recordings in your personal cloud storage bucket, you will still have the option to choose between zipped or unzipped M3U8 file uploads.
+> - If you're using 100ms storage, you cannot upload utilise the option of zipped M3U8 upload on any new templates.
```

Comment:
`If you're using 100ms storage, you cannot upload utilise the option of zipped M3U8 upload on any new templates.` -> `If you're selecting 100ms storage, you cannot choose the option of zipped VOD upload on any new templates.`

Reply:
Fixed